app/services/prosecution_case_recorder.rb

```diff
@@ -9,6 +9,13 @@ def initialize(prosecution_case_id, body)
 
   def call
     prosecution_case.update(body: body)
```

Comment:
I'm wondering if this can be taken off now?

Reply:
If I just remove it a load of tests fail. We're using the data in the `body` in a few places - for example in the `ProsecutionCase` model:  
```
def defendants
  body['defendants']&.map { |defendant| Defendant.new(body: defendant) }
end
```

Reply:
oh, yes, then not a good idea :+1: